### ISSUE TITLE
Add gradient norm to train output

### ISSUE THREAD

**Issue by twuebi** [stickeritis/sticker#54]
Printing the gradient norm alongside the training output can be helpful when tuning learning rates and for finding the gradient clipping threshold.

**Comment by danieldk** [comment]
The output is already a bit wide. I wonder if we really need the progress bar. It looks nice, but it takes up a lot of space and does not provide more information than just a percentage.

With the progress bar removed, we could add some more data points.

**Comment by twuebi** [comment]
I'm leaning towards already covered through Tensorboard. If we want to add it to the console: do we need a process bar or would a simple XY/YZ or XY% suffice?

**Comment by danieldk** [comment]
At least Tensorboard makes it possible to monitor a lot more things without things getting too crowded. In the terminal output I am typically only interested in the train/validation accuracy and sometimes loss. There already is a lot of visual information that make these kinda hard to spot. So now that you mention Tensorboard, maybe that is the best solution.

**Comment by twuebi** [comment]
That's already there for all models but the RNN.
